aten/src/ATen/native/Gesv.cpp

```diff
@@ -43,10 +43,16 @@ template<> void lapackGesv<double>(
 }
 #endif
 
+static inline bool isTransposeContiguous(Tensor& self) {
+ return self.dim() == 2 &&
+        self.stride(0) == 1 &&
+        self.stride(1) == self.size(0);
+}
+
 template <typename scalar_t>
 static void applyGesv(Tensor& b, Tensor& A, std::vector<int64_t> infos) {
```

Comment:
I think this was missed earlier, but `infos` has to be passed by reference to actually flag errors in `_gesv_helper_cpu`. This is not part of the PR, but I thought I should let you know.

Reply:
I found this is https://github.com/pytorch/pytorch/pull/9949 , and this change might break some tests for `gesv`.

Reply:
I'm confused. Isn't `info` being created and passed by reference below on line 63? `infos` is just accumulating the results for each batch of matrices, and we return from `applyGesv` as soon as we see an error.

Reply:
Yes @vishwakftw is correct. @animesht do you mind making this change? :)

Reply:
#10492